app/views/play/cinematic/Command/commands.js

```diff
@@ -40,3 +41,121 @@ export class Sleep extends AbstractCommand {
     return sleep(this.ms)
   }
 }
+
+/**
+ * AnimeCommand is used to turn animejs animation tweens into commands that the Command Runner can play and cancel.
+ */
+export class AnimeCommand extends AbstractCommand {
+  /**
+   * @param {anime} animation The animation that will be run.
+   */
+  constructor (animation) {
+    super()
+    this.animation = animation
+  }
+
+  /**
+   * Starts the animation, returning a cancellable promise that resolves when
+   * animation completes.
+   */
+  run () {
+    return new Promise((resolve, reject) => {
+      this.animation.play()
+      this.animation.complete = resolve
+    })
+  }
+
+  /**
+   * Cancel method ignores the promise and simply moves the animation
+   * to the end.
+   */
+  cancel (promise) {
+    const animation = this.animation
+    animation.seek(animation.duration)
+    return promise
+  }
+}
+
+/**
+ * SyncFunction runs a synchronous function.
+ */
+export class SyncFunction extends AbstractCommand {
+  /**
+   * @param {Function} runFn Synchronous function that doesn't return anything.
+   */
+  constructor (runFn) {
+    super()
+    this.run = () => {
+      runFn()
+      return Promise.resolve()
+    }
+    this.cancel = () => {}
+  }
+}
+
+export class Noop extends AbstractCommand {
+  run () {
+    return Promise.resolve()
+  }
+}
```

Comment:
Are there any changes in the code upto this or is it moved from AbstractCommand to here as it is?

Reply:
The `SequentialCommands` and `ConcurrentCommands` below are new. The `AnimeCommand`  is just moved. Sorry for not making that clearer.